### DISCUSSION TITLE
Spend Threshold worth of Kin

### DISCUSSION THREAD

**Discussion by Polype01** [kinfoundation/kin-rfcs#163] [KRE 4.0 Discussion]
Initially, the Spend Threshold worth of KIN (STwK) was $0.05 USD. I advocated for a much lower amount than initially proposed as higher STwK penalizes apps that have mostly transactions below that threshold (Example: social media apps/sites types). 
I like $0.01 USD as a starting point. 
Will that $0.01 USD STwK amount will evolve dynamically on a daily/weekly/monthly basis ?

**Comment by hann228** [comment]
Yes originally we contemplated a 2c spend increasing to 5c but based on community feedback (including yourself) we opted to start the amount at 1c.  At the moment there is no fixed plan to increase the amount beyond 1c.  The intention is to monitor the adoption of 1c spends and then look at any increase as a next step iteration.
